src/components/sdk/RosterAttendee/index.tsx

```diff
@@ -6,9 +6,10 @@ import React from 'react';
 import useAttendeeStatus from '../../../hooks/sdk/useAttendeeStatus';
 import { useRosterState } from '../../../providers/RosterProvider';
 import RosterCell, { RosterCellProps } from '../../ui/Roster/RosterCell';
+import { BaseSdkProps } from '../Base';
 import MicrophoneActivity from '../MicrophoneActivity';
 
-export interface RosterAttendeeProps extends Omit<RosterCellProps, 'name'> {
+export interface RosterAttendeeProps extends Omit<RosterCellProps, 'name'>, BaseSdkProps {
```

Comment:
Should this also extend `BaseProps` since `RosterCell` uses `BaseProps`? [Reference](https://github.com/aws/amazon-chime-sdk-component-library-react/blob/main/src/components/ui/Roster/RosterCell/index.tsx#L17-L18)

Reply:
No - the child component Props shouldnt dictate the parent component props - they should be independent of each other.

Reply:
You can still style the html elements inside of the RosterCell by passing a classname to the RosterAttendee and then applying styles to that classname.